web/routes.go

```diff
@@ -39,6 +40,12 @@ var routes = []route{
 		"/health",
 		healthHandler,
 	},
+	route{
+		"job_status",
```

Comment:
@sandlerben how should `route.Name` look? for `/add_job`, i called the route `initiateTranscriptionJob` vs. rachel's way `/job_status` route called `job_status` - which do you prefer?